pyproject.toml

```diff
@@ -16,23 +16,21 @@ classifiers = [
     "Topic :: Internet :: Proxy Servers",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Development Status :: 5 - Production/Stable",
-    "Programming Language :: Python :: 3.10",
```

Comment:
What was the reason to drop 3.10?
It is not EOL yet <https://devguide.python.org/versions/>
If we are dropping for features, why not drop 11 and 12?

Reply:
So, if it is a decision based on features, why keep 11 and 12?

Reply:
No good reason to keep it unless something needs it or old Python is being used in production on the server side.

It would make it a bit easier if 3.11 support went away as well.

Reply:
Even if some dependant needed older versions, they could just not update the package.
I think at least 11 can be removed, but probably also 12 also, I don't see why we need to keep old versions around when you can pin the lib to an older version.

Reply:
The only thing that would prevent pinning to and older version is the protocol needs to change a bit to support pause/resume to fix https://github.com/NabuCasa/snitun/issues/353 but in that case non-supported types would be ignored anyways and it won't break but the connection drop problem couldn't be fixed until both sides upgrade Python version/snitun versions but thats probably not a real concern.

I'll work on a PR to drop 3.11.

Reply:
https://github.com/NabuCasa/snitun/pull/354